WordPress/twentytwentyone
# Prevent link focus styles from affecting the buttons block

Fixes an issue where the link focus style is affecting the button block and the file download button

## Summary


Adds a :not for the button link and the file block download button.

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1. Add a button, make sure the button has a link.
1. Focus on the button.
1. Confirm that the link text has no underline, and that the background color for the link is not changed.



## Screenshots

## Quality assurance
